src/main/java/uk/gov/ons/ctp/response/lib/common/time/DateTimeUtil.java

```diff
@@ -31,6 +31,15 @@ public static Timestamp nowUTC() {
     return new Timestamp(System.currentTimeMillis());
   }
 
+  /**
+   * Non static method to retrieve the current time as a Timestamp.
+   *
+   * @return The current time in UTC
+   */
+  public Timestamp getNowUTC() {
```

Comment:
I don't quite get why this method has been implemented as non-static; this is a library class, it makes sense to call it in a static manner.

(Honestly, having a method just for `Timestamp(System.currentTimeMillis());` is weird but whatever)

Reply:
Removing this code from the CaseReceiptReceiver class and placing it in a non static method allows its returned value to be easily controlled (via mocking the DatetimeUtil class) within tests. It is created as a spring singleton so the overhead is negligible. The timestamp was previously passed in to the service, meaning that the tests could simply create the time, pass it to the service, and then check for that same time in the output. Now that the timestamp is created by the service, the tests can no longer control it unless they are able to mock out the code responsible for creating it - thus its placement in a non static method.